.github/workflows/code_coverage.yml

```diff
@@ -4,11 +4,13 @@ on:
   push:
     paths:
       - 'tests/**'
+      - '.github/workflows/**'
```

Comment:
What about changes to the source code? I assume these could trigger the code coverage action as well, even though not part of issue #35.

Reply:
That should of course trigger the action! You are right, this should be handled in another issue / branch.